src/js/background/messageHandler.js

```diff
@@ -187,7 +181,7 @@ const messageHandler = {
     // https://bugzil.la/1314674
     // https://github.com/mozilla/testpilot-containers/issues/608
     // ... so re-call displayBrowserActionBadge on window changes
-    badge.displayBrowserActionBadge();
+    badge.displayBrowserActionBadge("remove");
```

Comment:
This action is actually "showVersionIndicator". We (ab)use it to get around https://bugzilla.mozilla.org/show_bug.cgi?id=1314674 but it looks like that bug is fixed now? So we can just remove this call now.

Reply:
@groovecoder okay, so I should only focus on refactoring badge.displayBrowserActionBadge to handle 

1. when the user opens the 100th container and;
2. line 5 in badge.js

Reply:
thank you, I have effected the changes